storage/client/queries/queries.go

```diff
@@ -311,7 +311,11 @@ const (
 			txs.result_raw,
 			(
 				SELECT
-					json_object_agg(pre.address, encode(pre.address_data, 'hex'))
+					json_object_agg(pre.address, json_build_object(
```

Comment:
Why do we pull all this into Go? Does the consumer of this SELECT care about non-eth preimages? If not, I'd prefer to keep it simple and just add a `WHERE pre.context_identifier = ... AND pre.context_version = ...` here.

Reply:
Or rather, I should say: https://github.com/oasisprotocol/oasis-indexer/pull/368 pulls in just the eth preimages and discards other preimages. Is it OK to leave it that way, assuming I get the rest of the git merge right after this PR?

Reply:
it's nice to keep the preimage data together :shrug:

Reply:
will have to look at #368

Reply:
:facepalm: I just realized the whole column is called eth_addr_lookup to begin with

Reply:
Urgh I didn't think of this PR when I merged #368. Sorry I bulldozed it a little. I still think pulling the context is a bit of premature generalization, but yeah it can be nice in the future. Thanks for including the filtering on the DB side too, for now. And sorry for the rebase!

Reply:
:building_construction: